internal/pkg/bootstrap/handlers/database.go

```diff
@@ -139,7 +145,13 @@ func (d Database) BootstrapHandler(
 		},
 	})
 
-	lc.Info("Database connected")
+	var dbName string
+	if dbInfo.Type == postgresDBType {
+		dbName = "Postgres"
+	} else {
+		dbName = "Redis"
+	}
+	lc.Infof("%s Database connected", dbName)
```

Comment:
```suggestion
	lc.Infof("%s Database connected", dbInfo.Type)
```

the dbName variable may be confused with the real "database name" inside the database

Reply:
Done